Formula/spirv-tools.rb

```diff
@@ -0,0 +1,50 @@
+class SpirvTools < Formula
+  desc "API and commands for processing SPIR-V modules"
+  homepage "https://github.com/KhronosGroup/SPIRV-Tools"
+  url "https://github.com/KhronosGroup/SPIRV-Tools/archive/v2019.2.tar.gz"
+  sha256 "1fde9d2a0df920a401441cd77253fc7b3b9ab0578eabda8caaaceaa6c7638440"
+  depends_on "cmake" => :build
+
+  resource "re2" do
+    # revision number could be found in ./DEPS
+    url "https://github.com/google/re2.git",
+        :revision => "6cf8ccd82dbaab2668e9b13596c68183c9ecd13f"
+  end
+
+  resource "effcee" do
+    # revision number could be found in ./DEPS
+    url "https://github.com/google/effcee.git",
+        :revision => "04b624799f5a9dbaf3fa1dbed2ba9dce2fc8dcf2"
```

Comment:
Is there a good reason why you don't want to use git submodules or a CMake "SuperBuild" for this? Or provide these sources in a release tarball that is created every release to include the contents of the `external` directory?

Reply:
Since it is not a "git submodule" in the git repo, they use [gn](https://gn.googlesource.com/gn/) to sync commits between subprojects. See the [DEPS](https://github.com/KhronosGroup/SPIRV-Tools/blob/master/DEPS) file which is a part of gn build system.